Tests/FluentKitTests/FluentKitTests.swift

```diff
@@ -717,6 +717,18 @@ final class FluentKitTests: XCTestCase {
             .wait())
     }
     
+    func testPaginationDoesntCrashOnOverflow() throws {
+        let db = DummyDatabaseForTestSQLSerializer()
+        let pageRequest1 = PageRequest(page: 1184467440737095516, per: 1184467440737095516)
+        db.fakedRows.append([.init(["aggregate": 1])])
+        // This would crash on `Swift runtime failure: arithmetic overflow` if not handled
+        // so no point trying any XCTAssert
+        _ = try Planet2
+            .query(on: db)
+            .paginate(pageRequest1)
+            .wait()
+    }
+    
```

Comment:
It's true it will crash if it fails, but we can still assert that we get a predictable result from it:
```suggestion
    func testPaginationDoesntCrashOnOverflow() async throws {
        let db = DummyDatabaseForTestSQLSerializer()
        let pageRequest1 = PageRequest(page: 1184467440737095516, per: 1184467440737095516)
        db.fakedRows.append([.init(["aggregate": 1])])
        let result = try await Planet2
            .query(on: db)
            .paginate(pageRequest1)
        XCTAssertEqual(result.metadata.page, 1184467440737095516)
        XCTAssertEqual(result.metadata.per, 1184467440737095516)
        XCTAssertEqual(result.metadata.total, 1)
    }
    
```

Reply:
Sure, happy to do that